### PR TITLE
feat(v2): new docusaurus clear command

### PR DESCRIPTION
**Pull request by abadon7**
## Motivation
Implementing a new clear command to help users before reporting bugs as described in #3370 
### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan
yarn start:
No errors found.
Clear1

yarn clear:
Done message, no errors with the new command.
Clear2

System complaining because something is missing during run time
Clear4
Clear3

yarn start:
Everything is back to normal and working again.
Clear5
Clear6

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus/bin/docusaurus.js [hunk 1]
```diff
@@ -187,14 +188,21 @@ cli
     },
   );
 
+cli
+  .command('clear')
+  .description('Clean website')
```
Comment (lex111) [inline packages/docusaurus/bin/docusaurus.js:193]: I think this change will improve understanding of the CLI command.

```suggestion
  .description('Remove build artifacts')
```

**Review by lex111 — CHANGES_REQUESTED** [review]
Thank you for your contribution, but we still need to document the new command in the master docs.

**Comment by abadon7** [comment]
> Thank you for your contribution, but we still need to document the new command in the master docs.

Hi @lex111, is here where I should document the new command?, https://v2.docusaurus.io/docs/cli

**Comment by lex111** [comment]
> Hi @lex111, is here where I should document the new command?, https://v2.docusaurus.io/docs/cli

Yes, please.

**Comment by abadon7** [comment]
> > Hi @lex111, is here where I should document the new command?, https://v2.docusaurus.io/docs/cli
> 
> Yes, please.
Hi @lex111, is everything ok?, please, let me know is anything else is needed.

File: website/docs/cli.md [hunk 2]
```diff
@@ -126,3 +127,7 @@ Serve your built website localy.
 | `--dir` | `build` | The full path for the output directory, relative to the current workspace |
 | `--build` | `false` | Build website before serving |
 | `--host` | `localhost` | Specify a host to use. For example, if you want your server to be accessible externally, you can use `--host 0.0.0.0`. |
+
+### `docusaurus clear`
+
+Remove build artifacts.
```
Comment (lex111) [inline website/docs/cli.md:133]: Need to explain in more detail which folders are removed by this command.

**Review by lex111 — CHANGES_REQUESTED** [review]

**Comment by slorber** [comment]
Thanks

Did some little changes and added more doc, will merge when CI completes

**Comment by abadon7** [comment]
@slorber and @lex111 Thank you for supporting me and allowing me to contribute to this project. I'll start the hunt for more things that I can help with.

**Comment by slorber** [comment]
Thank you :) 

I've tagged a few starter issues, and guess some of them are still available

**Pull request merged**